docs/book/installation/sylius_plus_installation.rst

```diff
@@ -301,15 +301,15 @@ your application's ``.env`` file:
 
 .. code-block:: bash
 
-    bin/console sylius:install plus
+    bin/console sylius:install plus -s
 
 .. tip::
 
     If you want to completely (re)install the application, you can run this command with the no interaction flag ``-n``.
 
     .. code-block:: bash
 
-        bin/console sylius:install plus -n
+        bin/console sylius:install plus -s -n
```

Comment:
Should we install by default fixtures? The `-s` flag: `Load specified fixture suite during install`

Reply:
`s` was meant to be before `plus`. Good catch